### ISSUE TITLE
Function request: Convert_3DGS_to_Mesh_with_NeRF_and_Marching_Cubes List training parameters

### ISSUE THREAD

**Issue by liulsg** [MrForExample/ComfyUI-3D-Pack#103]
Convert_3DGS_to_Mesh_with_NeRF_and_Marching_Cubes has built-in training parameter 

nerf node, the higher the number of training steps, the better the effect, because you have built-in training steps, can you display the parameters of this node above?

**Comment by MrForExample** [comment]
Yes, that will be a part of my code refactoring & adding & optimization a few more better MVS algorithms with in next two weeks 👍
